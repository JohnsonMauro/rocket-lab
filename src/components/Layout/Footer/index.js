import Link from '@components/common/Form/Link';

import {
  FooterStyles,
  RightReserved,
  LinkContainer,
} from './styles';

const Footer = () => (
  <FooterStyles>
    <RightReserved>
      Â© 2020 Crunch Accounting. All right reserved.
    </RightReserved>
    <LinkContainer>
      <Link>Privacy Policy</Link>
      <Link>Terms of Service</Link>
    </LinkContainer>
  </FooterStyles>
);

export default Footer;
