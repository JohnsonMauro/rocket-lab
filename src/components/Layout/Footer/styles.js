import styled from '@emotion/styled';

const FooterStyles = styled.footer`
  align-items: center;
  display: flex;
  height: 50px;
  justify-content: space-between;
  padding: 0 166px;

  @media (max-width: 850px) {
    flex-direction: column;
    padding: 0 20px;
  }
`;

const RightReserved = styled.p`
  color: ${({ theme }) => theme.colors.blackRussian};
  font-size: 14px;
  line-height: 20px;
  opacity: 0.4;

  @media (max-width: 850px) {
    display: block;
    text-align: center;
    width: 100%;
  }
`;

const LinkContainer = styled.div`
  display: flex;
  justify-content: space-between;
  width: 216px;

  @media (max-width: 850px) {
    margin-bottom: 5px;
  }
`;

export { FooterStyles, RightReserved, LinkContainer };
