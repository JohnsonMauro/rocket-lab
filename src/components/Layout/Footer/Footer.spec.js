import React from 'react';
import { render } from '@testing-library/react';

import GlobalsContainer from '@test/GlobalsContainer';
import Footer from './index';

describe('<Footer />', () => {
  it('Renders the Footer component', () => {
    const { container } = render(<Footer />, {
      wrapper: GlobalsContainer,
    });
    expect(container).toMatchSnapshot();
  });
});
