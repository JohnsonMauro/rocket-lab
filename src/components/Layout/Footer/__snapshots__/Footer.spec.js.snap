// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Renders the Footer component 1`] = `
<div>
  <footer
    class="css-671999-FooterStyles e7d5c7x0"
  >
    <p
      class="css-1ab5tj5-RightReserved e7d5c7x1"
    >
      Â© 2020 Crunch Accounting. All right reserved.
    </p>
    <div
      class="css-1hwhkkb-LinkContainer e7d5c7x2"
    >
      <a
        class="css-1fwtyfg-LinkStyles e12g151z0"
        href="#"
      >
        Privacy Policy
      </a>
      <a
        class="css-1fwtyfg-LinkStyles e12g151z0"
        href="#"
      >
        Terms of Service
      </a>
    </div>
  </footer>
</div>
`;
